### PR TITLE
Remove double space in task template

### PR DESCRIPTION
**Pull request by kieranklaassen**
My rubocop was complaining about this double space every time so here is the easy fix! Thanks for this awesome Gem!!

### CODE REVIEW

File: lib/generators/task/templates/deploy.txt.erb [hunk 1]
```diff
@@ -8,5 +8,5 @@ namespace :after_party do
     # Update task as completed.  If you remove the line below, the task will
     # run with every deploy (or every time you call after_party:run).
     AfterParty::TaskRecord.create version: '<%= timestamp %>'
-  end  # task :<%= file_name %>
-end  # namespace :after_party
+  end # task :<%= file_name %>
```
Comment (boguth) [inline lib/generators/task/templates/deploy.txt.erb]: Hey I noticed here that you deleted a white space and I think that could seriously impair the functioning of our application. Let's write some specs to test if this is even feasible.

Comment (boguth) [inline lib/generators/task/templates/deploy.txt.erb]: Also 👍

**Comment by aehrmann** [comment]
+1

**Comment by kieranklaassen** [comment]
Any update?

**Comment by iiwo** [comment]
+1

**Comment by pelletencate** [comment]
+1

File: lib/generators/task/templates/deploy.txt.erb (hunk 1 above)
Comment (michaelirey) [reply lib/generators/task/templates/deploy.txt.erb]: Just curious, how could deleting this whitespace _"seriously impair the functioning of our application"_? Removing the whitespace here should have no impact, unless I really missing something.

Comment (boguth) [reply lib/generators/task/templates/deploy.txt.erb]: You're correct. This was my awful attempt at humor.

**Comment by theSteveMitchell** [comment]
Thanks @kieranklaassen, and apologies for the delay.

**Pull request merged**